pandas/tslib.pyx

```diff
@@ -2177,6 +2180,13 @@ cdef class _Timedelta(timedelta):
         """
         return timedelta(microseconds=int(self.value)/1000)
 
+cdef inline bint _has_ns(_Timedelta td):
+    """
```

Comment:
make this a method of ``_Timedelta``.

``self.value % 1000 != 0`` (might be same speed)